app/tasks/__init__.py

```
@@ -0,0 +1,3 @@
+# Import tasks so they can be discovered by Celery
+from app.tasks.process_document import process_document
```

Ruff (0.8.2)

`app.tasks.process_document.process_document` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)